crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1531,6 +1528,26 @@ unsafe impl<P: SystemParam + 'static> SystemParam for StaticSystemParam<'_, '_,
     }
 }
 
+// SAFETY: No world access.
+unsafe impl<T: ?Sized> SystemParam for PhantomData<T> {
+    type State = ();
+    type Item<'world, 'state> = Self;
```

Comment:
I *think* this is better than just returning `()` because it's easier to debug.

Reply:
It's also necessary, since `SystemParam`s have to be reflexive :)